### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by mgrimm84** [ladybug-tools/butterfly#253]
I am having problems installing OpenFOAM. I following the installation instruction's in Mostapha's YouTube video "01: How to install OpenFOAM for Windows" very carefully and scoured the web to find any help on my issue and didn't come up with anything. 

First thing - my system meets the requirements, virtualization has been turned on through BIOS. I am using Windows 7. When I open anything, I'm always selecting "Run as administrator."

The error starts when I try to open "Of_Env_Create". A terminal screen pops up very quickly and then goes away - too fast to even read what it says. I was able to finally read the error by opening it several times and take a picture of it by using the burst mode on my iPhone (I was very impressed with this method! ha!). Picture of error is below. Essentially, "Of_Env_Create" does not create a container becuase "'C:\Program Files (x86)\ESI\OpenFOAM\1706\Windows\Scripts\of_create_container.exe' is not recognized as an internal or external command, operable program or batch file." 

When I then try to open OpenFOAM, I get the following error: 
Error response from daemon: No such container: of_1706
Error: failed to start containers: of_1706

Both screenshots of the errors in Of_Env_Create and OpenFOAM_Start are below. Hopefully someone can help me out with this - I really would appreciate it!

Maria
![open foam create env](https://user-images.githubusercontent.com/30084563/28080968-952b12ac-6622-11e7-846e-b9410c6a3326.jpg)
![openfoam start](https://user-images.githubusercontent.com/30084563/28080993-a83e6dee-6622-11e7-9450-3507df3e85a3.PNG)

**Comment by mgrimm84** [comment]
One other note: It seems this has to do with the "of_create_container.exe" file....If I watch the ESI\OpenFOAM\1706\Windows\Scripts folder during the install.... "of_create_container.exe" shows up and then disappears. I'm not sure why. There are two other exe files that are installed that do not disappear. If someone that has a functional version of OpenFOAM working could tell me if they have the "of_create_container.exe" file in the Scripts folder that would be helpful to troubleshoot. Thank you.

**Comment by mgrimm84** [comment]
I reached out to OpenFOAM and Pawan Ghildiyal from ESI confirmed that the file "of_create_container.exe" was not installing properly for some reason. He was kind enough to write me a script to do what Of_Env_Create is supposed to do and now OpenFOAM is working properly. I have uploaded the script he wrote for me here in case anyone has the same issue. I am still not sure what was causing the problem, but thankfully I can not get OpenFOAM to work properly.

[containercreate.zip](https://github.com/ladybug-tools/butterfly/files/1145912/containercreate.zip)
